recipe/conda_build_config.yaml

```diff
@@ -82,6 +82,9 @@ target_gobin:
 # Rust Compiler Options
 rust_compiler:
   - rust
+# Pin rust to <1.49 on osx-64 due to min deployment target req, cf. https://github.com/conda-forge/rust-feedstock/issues/73
+rust_compiler_version:  # [osx and x86_64]
+  - 1.48                # [osx and x86_64]
```

Comment:
There's no 1.48 AFAIK. 1.46 maybe

Reply:
I didn't notice those were out of sync. Fixed, thanks.